danger_code_ws/1.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <cstdio>
+
+int main() {
+    char buffer[10];
+
+    std::cout << "Enter a string: ";
```

Comment:
The line `fgets(buffer, sizeof(buffer), stdin);` exceeds the recommended 80-character limit. Consider breaking it into multiple lines for better readability, or use a more concise method if possible.  While the guideline allows exceptions for string literals, this is a relatively short line; breaking it would improve readability.